com.unity.ml-agents/Runtime/AssemblyInfo.cs

```diff
@@ -1,6 +1,11 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Unity.ML-Agents.Editor.Tests")]
+[assembly: InternalsVisibleTo("Unity.ML-Agents.Runtime.Sensor.Tests")]
```

Comment:
Just curious, can we wildcard this to something like `InternalsVisibleTo("Unity.ML-Agents.*.Tests")`

Reply:
let me try

Reply:
looks like ['no'](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?redirectedfrom=MSDN&view=net-5.0#remarks)